### PR TITLE
FIX: device::get_info<max_work_group_size> = 1

### PR DESCRIPTION
**Pull request by JoanThibault**

### CODE REVIEW

File: include/CL/sycl/device.hpp [hunk 1]
```diff
@@ -208,9 +208,11 @@ class device
       \todo
   */
   template <info::device Param>
-  auto get_info() const {
+  size_t get_info() const {
     // Forward to the version where the info parameter is not a template
     //return get_info<typename info::param_traits_t<info::device, Param>>(Param);
+	assert(false); //non implemented
```
Comment (keryell) [inline include/CL/sycl/device.hpp]: This should be a static_assert instead.

File: tests/Makefile [hunk 2]
```diff
@@ -44,7 +44,7 @@ ifdef OpenCL_INCPATH
 	CXXFLAGS += -I$(OpenCL_INCPATH)
 endif
 
-# Specify where Bost.Compute is with BOOST_COMPUTE_INCPATH
+# Specify where Boost.Compute is with BOOST_COMPUTE_INCPATH
```
Comment (keryell) [inline tests/Makefile]: This commit is already in master as "fix a comment (#5)" https://github.com/keryell/triSYCL/commit/0f5a041f44bdfa6bd41d39acfc800d448b4a2525
So I have the feeling this commit should not be here.

File: include/CL/sycl/device.hpp [hunk 3]
```diff
@@ -245,7 +245,7 @@ class device
 
 template <>
 size_t device::get_info<info::device::max_work_group_size>() const {
```
Comment (keryell) [inline include/CL/sycl/device.hpp]: I think this should be "info::device::max_compute_units" if we want to restrict for simple local accessor for now, i.e. with only one work-group.

File: include/CL/sycl/device.hpp (hunk 1 above)
Comment (JoanThibault) [inline include/CL/sycl/device.hpp]: static_assert(false, "...") raises an error at compile blocking the compilation process.

Comment (JoanThibault) [inline include/CL/sycl/device.hpp]: static_assert(false, "...") fails at compile time even if no call to this function (even if this function is template) is done.

**Comment by keryell** [comment]
Thank you.

**Pull request merged**